app/views/candidate_interface/subject_knowledge/show.html.erb

```diff
@@ -5,11 +5,25 @@
   <%= t('page_titles.subject_knowledge') %>
 </h1>
 
+<% if @application_form.review_pending?(:subject_knowledge) %>
+  <div class='govuk-inset-text govuk-!-width-two-thirds govuk-!-margin-top-0'>
```

Comment:
I think we can use the `GovukComponent::InsetText` component here, now that it supports block content?

Reply:
nice! 👍🏼